sources/constraint-definition.asciidoc

```diff
@@ -436,7 +436,7 @@ If left undefined, the default value for [methodname]`@OverridesAttribute.name`
 
 [NOTE]
 ====
-[tck-testable]#A composing constraint can itself be a composed constraint. In this case, attribute values are overridden recursively according to the described rules.# Note however, that a forwarding rule (as defined by [classname]`@OverridesAttribute`) is only applied to the direct composing constraints.
```

Comment:
Wasn't this correct? A constraint given as meta-annotation (=composing constraint) can be composed from others itself.